INSTALL.md

```diff
@@ -1 +1,114 @@
+ETW exporter need nlohmann-json to build. This is covered in the build
+instructions for these components.
+
+### Building as Standalone CMake Project
+
+1. Getting the opentelementry-cpp source:
+
+   ```console
+   # Change to the directory where you want to create the code repository
+   $ cd ~
+   $ mkdir source && cd source
+   $ git clone https://github.com/open-telemetry/opentelemetry-cpp
+   Cloning into 'opentelemetry-cpp'...
+   ...
+   Resolving deltas: 100% (3225/3225), done.
+   $
+   ```
+
+2. Navigate to the repository cloned above, and create the `CMake`
+build configuration.
+
+   ```console
+   $ cd opentelemetry-cpp
+   $ mkdir build && cd build
+   $ cmake .. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_CXX_FLAGS="-Werror"
+   -- The C compiler identification is GNU 9.3.0
+   -- The CXX compiler identification is GNU 9.3.0
+   ...
+   -- Configuring done
+   -- Generating done
+   -- Build files have been written to: /home/<user>/source/opentelemetry-cpp/build
+   $
+   ```
+
+3. Once build configuration is created, build the CMake targets -
+ this includes building SDKs, and building unittests for API and SDK.
+ Note that since API is header only library, no separate build is triggered for it.
+
+   ```console
+   $ cmake --build . --target all
+   Scanning dependencies of target timestamp_test
+   [  0%] Building CXX object api/test/core/CMakeFiles/timestamp_test.dir/timestamp_test.cc.o
+   [  1%] Linking CXX executable timestamp_test
+   ...
+   Scanning dependencies of target w3c_tracecontext_test
+   [ 99%] Building CXX object ext/test/w3c_tracecontext_test/CMakeFiles/w3c_tracecontext_test.dir/main.cc.o
+   [100%] Linking CXX executable w3c_tracecontext_test
+   [100%] Built target w3c_tracecontext_test
+   $
+   ```
+
+4. Once CMake tests are built, run them with `ctest` command
+
+   ```console
+   $ ctest
+   Test project /tmp/opentelemetry-cpp/build
+        Start   1: trace.SystemTimestampTest.Construction
+   ...
+        Start 380: ext.http.urlparser.UrlParserTests.BasicTests
+   ...
+   100% tests passed, 0 tests failed out of 380
+   $
+   ```
+
+5. Optionally install the header files for API, and generated
+targets and header files for SDK at custom/default install location.
+
+   ```console
+   $ cmake --install . --config Debug --prefix /<install_root>/
+   -- Installing: /<install-root>/lib/cmake/opentelemetry-cpp/opentelemetry-cpp-config.cmake
+   -- Installing: /<install-root>/lib/cmake/opentelemetry-cpp/opentelemetry-cpp-config-version.cmake
+   ...
+   -- Installing: /<install-root>/include/opentelemetry//ext/zpages/static/tracez_index.h
+   -- Installing: /<install-root>/include/opentelemetry//ext/zpages/static/tracez_style.h
+   -- Installing: /<install-root>/include/opentelemetry//ext/zpages/threadsafe_span_data.h
+   -- Installing: /<install-root>/lib/libopentelemetry_zpages.a
+   $
+   ```
+
+### Incorporating Into An Existing CMake Project
+
+TBD
```

Comment:
Perhaps we can describe the same method as is used by nginx instrumentation:
```
find_package(opentelemetry-cpp REQUIRED)
find_package(gRPC REQUIRED)
target_include_directories(my_target PRIVATE ${OPENTELEMETRY_CPP_INCLUDE_DIRS})
target_link_libraries(my_target PRIVATE ${OPENTELEMETRY_CPP_LIBRARIES} gRPC::grpc++)
```

Reply:
I was thinking to keep it as part of #564 fix. But if it's already used in nginx, let me go though that once. Thanks for sharing the configuration here.

Reply:
It's done now.